manifests/0000_90_dns-operator_03_prometheusrules.yaml

```diff
@@ -34,3 +34,15 @@ spec:
           severity: warning
         annotations:
           message: "CoreDNS is returning SERVFAIL for {{ $value | humanizePercentage }} of requests."
+      - alert: CoreDNSForwardHealthcheckFailureCount
+        annotations:
+          message: '{{ $value }} A CoreDNS counter for the number of failed health checks for an upstream server on {{ $labels.instance }}'
+        expr: coredns_forward_healthcheck_failure_count_total > 0
```

Comment:
These alerts will fire until the applicable CoreDNS instances are restarted, since the counter is monotonically increasing.
I am in favor of using `increase(<metric>) > 0` instead, so that the alert is not continuously firing after the CoreDNS issues have been resolved (Which may or may not involve CoreDNS pods being restarted). 
The `CoreDNSPanicking` alert in this file already follows this pattern. Thoughts?

Reply:
Here we want to do an error rate based alert. Here are some examples:

https://github.com/openshift/cluster-monitoring-operator/blob/09180fe1a032cf9fe0c0d98d1feb35c3ee2c21a0/assets/thanos-ruler/thanos-ruler-prometheus-rule.yaml#L32-L37

https://github.com/coreos/kube-prometheus/blob/master/manifests/prometheus-rules.yaml#L1853-L1868